test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -171,15 +195,20 @@ func TestCtlCreateCR(t *testing.T) {
 				}
 			}
 
+			// Try to set Ready Condition if needed, otherwise the test just times out
```

Comment:
Could you update this comment to be a bit more specific, e.g.

```
// Start a goroutine that will periodically check whether the CertificateRequest has been created by the CLI command yet.
// Once it has been created, set the `status.certificate` and `Ready` condition so that the `--fetch` part of the
// command is able to proceed.
```

I had to read this 2 or 3 times to spot the `go` bit 🙈